lib/scutil/pgnewfil.c

```diff
@@ -32,6 +33,10 @@ int pgnewfil_debug = 0;
 #endif
 extern size_t strlen();
 
+#ifndef S_ISDIR 
+#define S_ISDIR(mode)  (((mode) & S_IFMT) == S_IFDIR) 
```

Comment:
As far as I see in stat.h S_IFMT and S_IFDIR refer to underscored version of them, if _CRT_INTERNAL_NONSTDC_NAMES is defined. Should we use _S_IFMT and _S_IFDIR instead?